GIFrameworkMaps.Web/Views/ManagementTheme/Edit.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@model GIFrameworkMaps.Data.Models.Theme
+@{
+    ViewData["Title"] = $"Edit theme '{Model.Name}'";
+}
+
+<h1>@ViewData["Title"]</h1>
+<div class="row mt-2 mb-2">
+    <div class="col-md-8">
+        <form asp-action="Edit">
+            <div asp-validation-summary="ModelOnly" class="alert alert-danger mb-3"></div>
+            <input type="hidden" asp-for="Id" />
+            <div class="mb-3">
+                <label asp-for="Name" class="control-label"></label>
+                <input asp-for="Name" class="form-control" />
+                <span asp-validation-for="Name" class="text-danger"></span>
+            </div>
+            <div class="mb-3">
+                <label asp-for="Description" class="control-label"></label>
+                <textarea asp-for="Description" class="form-control"></textarea>
+                <span asp-validation-for="Description" class="text-danger"></span>
+            </div>
+            <div class="mb-3">
+                <label asp-for="PrimaryColour" class="control-label"></label>
+                <input asp-for="PrimaryColour" class="form-control form-control-color" type="color" />
```

Comment:
The primary colour isn't being shown in the picker when you edit a theme, as the data that comes in here is the hex code without the '#' (as it is stored in the database), but `<input type="color" />` requires the '#'. We'll need to make sure it is added when this input is rendered (or re-do how it is stored in the database to use the '#', but I think there was a reason we didn't include the '#'...)

Reply:
Fixed